Sources/SwiftDocC/Semantics/MarkupReferenceResolver.swift

```diff
@@ -26,6 +26,11 @@ fileprivate func unknownSnippetSliceProblem(snippetPath: String, slice: String,
     return Problem(diagnostic: diagnostic, possibleSolutions: [])
 }
 
+fileprivate func removedLinkDestinationProblem(reference: ResolvedTopicReference, source: URL?, range: SourceRange?, severity: DiagnosticSeverity) -> Problem {
+    let diagnostic = Diagnostic(source: source, severity: severity, range: range, identifier: "org.swift.docc.removedExtensionLinkDestination", summary: "The topic \(reference.path.singleQuoted) is an empty extension page and cannot be linked to.", explanation: "This extension symbol has had all its children curated and has been removed.")
+    return Problem(diagnostic: diagnostic, possibleSolutions: [])
```

Comment:
I _think_ this would be an easy `Solution` to replace the entire source range of the link with part of the reference's path wrapped in <code>`</code>. 

IIRC the first to path components of the resolver reference will always be "/documentation/<ExtendingModule>" so we'd would skip those and only suggest the path from the 3rd path component, which should be the extended module.

Reply:
You were right; the given range points to the full link and double-backticks, so replacing it with an equivalent text in single-backticks was straightforward.